src/main/kotlin/no/digdir/fdk/searchservice/mapper/DatasetMapper.kt

```diff
@@ -0,0 +1,23 @@
+package no.digdir.fdk.searchservice.mapper
+
+import no.digdir.fdk.searchservice.model.*
+
+fun Dataset.toSearchObject() =
+    SearchObject(
+        id = id,
+        uri = "",
+        accessRights = accessRights,
+        catalog = null,
+        dataTheme = theme,
+        description = description,
+        fdkFormat = null,
```

Comment:
lag en metode som trekker ut og prefikser alle format og mediatyper fra datasett
```suggestion
        fdkFormatPrefixed = prefixedFormats(),
```

Reply:
fikset noen av disse feltene. Men tar andre verdien i eget issue.
https://github.com/Informasjonsforvaltning/fdk-search-service/issues/72